### ISSUE TITLE
Multi-Label Image Classication

### ISSUE THREAD

**Issue by jtlz2** [scikit-multilearn/scikit-multilearn#90]
Apologies for such a simple question: Can (and how) can I use scikit-multilearn on images (probably JPEGs)? Thanks!

**Comment by ChristianSch** [comment]
Hi there, no problem!

For an example please see [the sklearn docs](http://scikit-learn.org/stable/auto_examples/classification/plot_digits_classification.html) here. The basic idea is that we have a 2d image with 3 data points per pixel (RGB color) or binary for black/white or even gray scale. The idea then is to flatten the image, such that we obtain a simple 1d vector for a tensor. This of course removes some information about the features, but I actually don't know what models can work with that, except than convolutional neural networks.

The relevant code in the link above:
```
# To apply a classifier on this data, we need to flatten the image, to
# turn the data in a (samples, feature) matrix:
n_samples = len(digits.images)
data = digits.images.reshape((n_samples, -1))
```

**Comment by jtlz2** [comment]
@ChristianSch Many thanks for the speedy reply. Do you think this would be feasible for images that are few 100 x few 100 pixels (the digits example is rather smaller)? This is starting to sound like a heavy-lift operation, which perhaps can't be avoided. Thanks again.

**Comment by ChristianSch** [comment]
Hm, I don't normally work with images. I guess it depends if its full blown rgb and what hardware you have. You can always decrease the size of the images.

**Comment by niedakh** [comment]
@jtlz2 in short - if you want to do image classification flatteing them is the simplest approach, if size differ - you should you sparse matrices as input - a row shoud reflect an image, a column - the number of a pixel. it is the simplest approach.  the scene dataset is basically that. 

you can also use sift or other visual bag of words approaches, see https://ianlondon.github.io/blog/how-to-sift-opencv/

i've added an issue to add an example with this
